### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by RogueGhost93** [qdm12/gluetun#1603]
### Is this urgent?

Yes

### CPU arch

x86_64

### VPN service provider

Mullvad

### What are you using to run the container

docker-compose

### What is the version of Gluetun

latest

### What's the problem 🤔

Gluetun keeps saying the server value is not on the list but it actually is. I tried with a few countries and the same thing happens. 

Im usually using UK and it also doesnt work. I tried using with caps lock and without. Very weird, it was working fine until today and i cant figure out what the issue is. 

### Share your logs

```plain text
2023-05-25T19:00:31Z ERROR VPN settings: provider settings: server selection: for VPN service provider mullvad: the country specified is not valid: value is not one of the possible choices: value usa, choices available are Albania, Australia, Austria, Belgium, Brazil, Bulgaria, Canada, Colombia, Croatia, Czech Republic, Denmark, Estonia, Finland, France, Germany, Greece, Hong Kong, Hungary, Ireland, Israel, Italy, Japan, Latvia, Luxembourg, Moldova, Netherlands, New Zealand, North Macedonia, Norway, Poland, Portugal, Romania, Serbia, Singapore, Slovakia, South Africa, Spain, Sweden, Switzerland, UK, USA, United Arab Emirates


2023-05-25T19:03:35Z ERROR VPN settings: provider settings: server selection: for VPN service provider mullvad: the country specified is not valid: value is not one of the possible choices: value croatia, choices available are Albania, Australia, Austria, Belgium, Brazil, Bulgaria, Canada, Colombia, Croatia, Czech Republic, Denmark, Estonia, Finland, France, Germany, Greece, Hong Kong, Hungary, Ireland, Israel, Italy, Japan, Latvia, Luxembourg, Moldova, Netherlands, New Zealand, North Macedonia, Norway, Poland, Portugal, Romania, Serbia, Singapore, Slovakia, South Africa, Spain, Sweden, Switzerland, UK, USA, United Arab Emirates
```

**Comment by oester** [comment]
Same issue here - I was using Toronto, and that's in the list...

**Comment by Gorian** [comment]
Same issue here - I even copied and pasted from the list it output

**Comment by liamrpowell** [comment]
Same issue here. Using USA.

**Comment by Fastjur** [comment]
Hi all, please don't comment you have the same issue, it will only add to many emails. Just thumbs up the original comment.

Pretty certain that this commit: https://github.com/qdm12/gluetun/commit/a43973c093662c2b5521eb2e9e3aea053584e96b#diff-e614e18f3fe5d5bdbace977e5feeb42a34c6fd5adbcd469c8cd2ed161acfc885
with a move to `gosettings` has made reading the env ignore case.

**Comment by qdm12** [comment]
The latest image `qmcgaw/gluetun` was fixed a few minutes ago by 7e325715c7a24da73447b29cec94b207bd950745.
The regression was introduced 2 days ago with a43973c093662c2b5521eb2e9e3aea053584e96b which did compare text taking care about case sensitivity (gluetun lowercases everything inputted, but keeps servers data text with case sensitivity). My apologies for the delay fixing it, obviously, and thank you for reporting the issue 🎖️ 

Just in case you are not aware, you can use released docker images such as `qmcgaw/gluetun:v3` or `qmcgaw/gluetun:v3.34.1` which should be more stable than the latest image `qmcgaw/gluetun`. I always appreciate users using the latest image since this helps fix bugs before bringing them to a stable release which would end up unstable, so ideally keep using the latest image 😉
